addons/l10n_it_edi_withholding/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Italy - E-invoicing (Withholding)',
+    'version': '0.1',
+    'depends': [
+        'l10n_it_edi'
+    ],
+    'author': 'Odoo',
+    'description': """
+Withholding and Pension Fund handling for the E-invoice implementation for Italy.
+
+    The Withholding tax and the Pension Fund tax are computed like every other tax
+    with the ordering by sequence, so please check that your tax configuration is correct.
+    """,
+    'category': 'Accounting/Localizations/EDI',
+    'website': 'http://www.odoo.com/',
```

Comment:
Do we have a documentation url?

Reply:
Documentation link?
https://www.odoo.com/documentation/16.0/applications/finance/accounting/fiscal_localizations/localizations/italy.html
But it has to change with every version... ?